MycroftAI/mycroft-core
# Add noise_words.list for Swedish

## Description
This adds noise words for exclusion when calculating the best query response.

## How to test
Check that CQS skills can be loaded or if the fix for the skill load failure is merged that it doesn't give a warning.

## Contributor license agreement signed?
CLA [X]
